code/messaging/src/phone/java/com/adobe/marketing/mobile/Messaging.java

```diff
@@ -180,15 +180,6 @@ public static void refreshInAppMessages() {
                 .setEventData(eventData)
                 .build();
 
-        MobileCore.dispatchEventWithResponseCallback(refreshMessageEvent, TIMEOUT_MILLIS, new AdobeCallbackWithError<Event>() {
```

Comment:
removed the call to dispatchEventWithResponseCallback as the error callback wasn't actually thrown in the messaging code. this would lead to the error callback being called after a timeout elapsed.

Reply:
Good catch!